src/context/OpenAIProvider.tsx

```diff
@@ -243,12 +261,14 @@ export default function OpenAIProvider({ children }: PropsWithChildren) {
       messages_ = messages_.length ? messages_ : messages;
 
       try {
+        // FIXME: try to get easy auth token every time
```

Comment:
Adhoc implementation for now, since ...

- frontend app sends request to `/.auth/me` to retrieve JWT token every time before submitting chat message, which is necessary only once theoretically
- no need to sent jwt, if EasyAuth is not enabled

Reply:
> no need to sent jwt, if EasyAuth is not enabled

To branch behaviors, I tried to use `NEXT_PUBLIC_*` variables, but they are expanded on building the frontend app...